.buildkite/pipeline.full.yml

```diff
@@ -242,6 +244,8 @@ steps:
           - "--fail-fast"
     concurrency: 9
     concurrency_group: 'browserstack-app'
+    soft_fail:
```

Comment:
Please comment these to make it clear that we're working around a temporary browserstack issue (presumed) and that the soft fail should be removed as soon as possible once confirmed and fixed.